### PR TITLE
Home page content sections

### LINKED ISSUE #2: Create home page
Home page :
Is a description page.

- [x] Create a header section in HTML and its style. 
- [x] Create navbar bar in html and its style. 
- [x] Create showcase section in html and its style.
- [x] Create News section in html and its style.
- [x] Create footer section in html and its style.
- [x] Make the page responsive

### PR DESCRIPTION
**Pull request by ahmedhamouda2**
add everything related to content sections on the home page

### CODE REVIEW

File: home-page/style.css [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+a {
+    color: #000;
+    text-decoration: none;
+}
+/* start section showcase */
+.showcase-fluid {
+    background-image: url('../images/home/background-showcase.jpg');
+    background-size: cover;
+    background-position:center;
+    background-repeat: no-repeat;
```
Comment (Mu7ammadAbed) [inline home-page/style.css]: You can try something like this and you can even increase the black transparent background
```suggestion
    background: linear-gradient(to bottom, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)),
    url("../images/home/background-showcase.jpg") no-repeat center center;
    background-size: cover;
```

Comment (ahmedhamouda2) [reply home-page/style.css]: I did this now

**Review by NadiaJalalTamimi — APPROVED** [review]
Greaaaaaat

**Review by omar-shaath — COMMENTED** [review]
Great job as usual!

**Review by omar-shaath — APPROVED** [review]
Great job as usual!

**Review by NourAdawi — APPROVED** [review]

**Pull request merged**